### PR TITLE
fix(categoryOptionGroupSet): update view config and formName validation

### PR DESCRIPTION
**Pull request by Chisomchima**

### CODE REVIEW

File: src/lib/sectionList/listViews/sectionListViewsConfig.ts [hunk 1]
```diff
@@ -59,7 +59,6 @@ export const defaultModelViewConfig = {
             'createdBy',
             'href',
             'id',
-            'lastUpdatedBy',
```
Comment (Birkbjo) [inline src/lib/sectionList/listViews/sectionListViewsConfig.ts:62]: Why is this removed? This is default and show be available for all models. If not, it can be overridden for a specific model with `overrideDefaultAvailable`.

**Review by Birkbjo — APPROVED** [review]
LGTM. Can you update the PR title to be a bit more specific. Eg.

```fix(categoryOptionGroupSet): update view config and formName validation```

**Review by flaminic — APPROVED** [review]
